app/models/operatingsystem.rb

```diff
@@ -46,6 +46,12 @@ class Operatingsystem < ApplicationRecord
 
   default_scope -> { order(:title) }
 
+  scope :sort_by_version, lambda {
+    to_a.sort_by do |os|
+      Gem::Version.new("#{os.major}.#{os.minor}")
```

Comment:
This doesn't deal with operating systems that don't have a minor and there's already a method that does that concatenation.
```suggestion
      Gem::Version.new(os.release)
```

Reply:
> This doesn't deal with operating systems that don't have a minor

Yes, that's true, right now it's heavily `minor` version oriented. During the POC, I didn't want to include `major` versions in the scope, but the more I think about it, the more I'm considering why not?